spec/unit/build_task_spec.rb

```diff
@@ -44,6 +44,7 @@
 
   describe "#build" do
     it "stores the kiwi config to a temporary directory" do
+      allow(LocalSystem).to receive(:validate_existence_of_packages).and_return("")
```

Comment:
The result of the call is not checked, so it's not needed to return anything here. Same for the other calls.